unable to install from npm

On OSX

With npm 7.21.0

```
⚡  ~/code/metaplex/test_semantic_release npm install @metaplex/js
npm WARN deprecated circular-json@0.5.9: CircularJSON is in maintenance only, flatted is its successor.
npm ERR! code 127
npm ERR! path /Users/aaron/code/metaplex/test_semantic_release/node_modules/@metaplex/js
npm ERR! command failed
npm ERR! command sh -c husky install
npm ERR! sh: husky: command not found

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/aaron/.npm/_logs/2022-01-07T02_01_28_969Z-debug.log
```

With yarn 1.22.17

```
⚡  ~/code/metaplex/test_semantic_release yarn add @metaplex/js
yarn add v1.22.13
warning ../../package.json: No license field
info No lockfile found.
[1/4] 🔍  Resolving packages...
warning @metaplex/js > @solana/web3.js > rpc-websockets > circular-json@0.5.9: CircularJSON is in maintenance only, flatted is its successor.
[2/4] 🚚  Fetching packages...
[3/4] 🔗  Linking dependencies...
warning " > @metaplex/js@4.11.1" has unmet peer dependency "@metaplex-foundation/mpl-auction@^0.0.2".
warning " > @metaplex/js@4.11.1" has unmet peer dependency "@metaplex-foundation/mpl-core@^0.0.2".
warning " > @metaplex/js@4.11.1" has unmet peer dependency "@metaplex-foundation/mpl-metaplex@^0.0.5".
warning " > @metaplex/js@4.11.1" has unmet peer dependency "@metaplex-foundation/mpl-token-metadata@^0.0.2".
warning " > @metaplex/js@4.11.1" has unmet peer dependency "@metaplex-foundation/mpl-token-vault@^0.0.2".
warning " > @metaplex/js@4.11.1" has unmet peer dependency "@solana/spl-token@^0.1.8".
warning " > @metaplex/js@4.11.1" has unmet peer dependency "@solana/web3.js@^1.30.2".
[4/4] 🔨  Building fresh packages...
[-/4] ⠂ waiting...
[2/4] ⠂ utf-8-validate
[3/4] ⠂ secp256k1
error /Users/aaron/code/metaplex/test_semantic_release/node_modules/@metaplex/js: Command failed.
Exit code: 127
Command: husky install
Arguments:
Directory: /Users/aaron/code/metaplex/test_semantic_release/node_modules/@metaplex/js

--- a/package.json
+++ b/package.json
@@ -40,7 +40,7 @@
     "test:all": "yarn test:build && yarn test:init && yarn test",
     "test": "jest",
     "test:watch": "jest --watch",
-    "postinstall": "husky install",
+    "prepare": "husky install",
     "link-mpl": "yarn link ../metaplex-program-library --private -r -A"
   },
   "dependencies": {

